packages/scheduler/src/forks/SchedulerHostConfig.post-task.js

```diff
@@ -5,246 +5,117 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import {enableIsInputPending} from '../SchedulerFeatureFlags';
+// Capture local references to native APIs, in case a polyfill overrides them.
+const date = window.Date;
+const perf = window.performance;
+const setTimeout = window.setTimeout;
+const clearTimeout = window.clearTimeout;
+
+// This check should be done upstream but do it again for a clear failure message.
+if (global.scheduler === undefined || global.scheduler.postTask === undefined) {
+  throw new Error('Cannot use postTask Scheduler without global.scheduler');
+}
 
-export let requestHostCallback;
-export let cancelHostCallback;
-export let requestHostTimeout;
-export let cancelHostTimeout;
-export let shouldYieldToHost;
-export let requestPaint;
-export let getCurrentTime;
-export let forceFrameRate;
+function postTask(callback) {
+  // Use experimental Chrome Scheduler postTask API.
+  global.scheduler.postTask(callback);
+}
 
-if (
-  // If Scheduler runs in a non-DOM environment, it falls back to a naive
-  // implementation using setTimeout.
-  typeof window === 'undefined' ||
-  // Check if MessageChannel is supported, too.
-  typeof MessageChannel !== 'function'
-) {
-  // If this accidentally gets imported in a non-browser environment, e.g. JavaScriptCore,
-  // fallback to a naive implementation.
-  let _callback = null;
-  let _timeoutID = null;
-  const _flushCallback = function() {
-    if (_callback !== null) {
-      try {
-        const currentTime = getCurrentTime();
-        const hasRemainingTime = true;
-        _callback(hasRemainingTime, currentTime);
-        _callback = null;
-      } catch (e) {
-        setTimeout(_flushCallback, 0);
-        throw e;
-      }
-    }
-  };
-  const initialTime = Date.now();
-  getCurrentTime = function() {
-    return Date.now() - initialTime;
-  };
-  requestHostCallback = function(cb) {
-    if (_callback !== null) {
-      // Protect against re-entrancy.
-      setTimeout(requestHostCallback, 0, cb);
-    } else {
-      _callback = cb;
-      setTimeout(_flushCallback, 0);
-    }
-  };
-  cancelHostCallback = function() {
-    _callback = null;
-  };
-  requestHostTimeout = function(cb, ms) {
-    _timeoutID = setTimeout(cb, ms);
-  };
-  cancelHostTimeout = function() {
-    clearTimeout(_timeoutID);
-  };
-  shouldYieldToHost = function() {
-    return false;
-  };
-  requestPaint = forceFrameRate = function() {};
+let getNow;
+if (typeof perf === 'object' && typeof perf.now === 'function') {
+  getNow = () => perf.now();
```

Comment:
Same here. Any browser that supports `postTask`  will also support `performance.now`.